src/ops/ops.go

```diff
@@ -82,6 +82,18 @@ func (o *ops) ExecCommand(liveLogger io.Writer, command string, args ...string)
 		cmd.Stdout = &stdoutBuf
 		cmd.Stderr = &stdoutBuf
 	}
+	if config.GlobalConfig.HTTPProxy != "" || config.GlobalConfig.HTTPSProxy != "" {
+		cmd.Env = os.Environ()
```

Comment:
Consider calculating the proxy env once in the ops builder and just appending it to the cmd.env here